plugin/mutate/source/dextool/plugin/mutate/backend/database/type.d

```diff
@@ -225,3 +225,17 @@ struct MutantMetaData {
         attr.match!((NoMetadata a) {}, (NoMut a) => put(w, "nomut"));
     }
 }
+
+struct Rationale {
+  string time;
+  string rationale;
+}
+
+// Mutant manually marked in db
+struct MarkedMutant {
+    MutationId mutant_id;
```

Comment:
This should be a mutation status id.
The mutationId will change each analyze thus it becomes invalid almost directly when used in practise.
It should also be marked as the primary key via the necessary UDA.